### PR TITLE
chore: remove Node.js 4, add Node.js 10

### PR DESCRIPTION
**Pull request by DanielRuf**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -8,10 +8,10 @@ environment:
     - nodejs_version: '8'
       webpack_version: latest
       job_part: test
-    - nodejs_version: '6'
+    - nodejs_version: '8'
       webpack_version: latest
       job_part: test
-    - nodejs_version: '4.3'
+    - nodejs_version: '6'
```
Comment (evilebottnawi) [inline appveyor.yml]: Don't remove `node@4`, it will be remove when we drop node@4 support, just add `node@10`

File: .travis.yml [hunk 2]
```diff
@@ -23,13 +23,17 @@ jobs:
...
+      env: WEBPACK_VERSION=latest JOB_PART=test
+      script: npm run travis:$JOB_PART
     - stage: Webpack canary
       before_script: yarn add git://github.com/webpack/webpack.git#master
       script: npm run travis:$JOB_PART
       node_js: 8
       env: WEBPACK_VERSION=canary JOB_PART=test
 before_install:
-  - 'if [[ `npm -v` != 5* ]]; then npm i -g npm@^5.0.0; fi'
+  - npm i -g npm@latest
```
Comment (evilebottnawi) [inline .travis.yml]: npm@6 don't support node 4 left this as is

Comment (DanielRuf) [reply .travis.yml]: node 4 is EOL and fails (see the failing unit tests of the last commits in the master branch).

File: appveyor.yml (hunk 1 above)
Comment (DanielRuf) [reply appveyor.yml]: Sure? See the last commits and the failing test suite. The dependencies require node >= 6.

Comment (DanielRuf) [reply appveyor.yml]: https://circleci.com/gh/webpack-contrib/mini-css-extract-plugin/29

**Comment by DanielRuf** [comment]
We can not test further before this is not fixed: https://github.com/webpack-contrib/mini-css-extract-plugin/issues/129

Not sure where the config for CircleCI is and how it is currently configured but it seems it is not correctly working.

**Comment by evilebottnawi** [comment]
@DanielRuf should be works now, can you rebase?

**Comment by DanielRuf** [comment]
Thanks for the update. Will do so later today.

**Comment by DanielRuf** [comment]
> @DanielRuf should be works now, can you rebase?

So basically these files are not there anymore so I close this PR.

**Pull request closed without merging**